force-app/main/default/lwc/nksEmbedGenerator/nksEmbedGenerator.js

```diff
@@ -19,16 +19,27 @@ export default class NksEmbedGenerator extends LightningElement {
                 console.error(JSON.stringify(error, null, 2));
             });
     }
-
+    
     handleCopy() {
-        navigator.clipboard.writeText(this.embeddingCode);
-        this.showCopyToast();
+        let copyValue = this.embeddingCode;
+        let hiddenInput = document.createElement('input');
+        hiddenInput.value = copyValue;
+        document.body.appendChild(hiddenInput);
+        hiddenInput.focus();
+        hiddenInput.select();
+        try {
+            var successful = document.execCommand('copy');
+            this.showCopyToast(successful ? 'success' : 'error');
+        } catch (error) {
+            this.showCopyToast('error');
+        }
+        document.body.removeChild(hiddenInput);
```

Comment:
Du burde også sette fokus tilbake til knappen som ble trykket. Ellers så blir fokus satt til toppen/bunnen (tror det er bunnen) av skjermen for skjermlesere og de må navigere seg tilbake igjen. (Dette er jeg ikke 100% sikker på da, men jeg tror det fungerer sånn hehe)